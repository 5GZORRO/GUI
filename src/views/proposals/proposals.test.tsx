// @ts-nocheck
import React from 'react'
import TestRenderer from 'react-test-renderer'
import Proposals from './index'

describe('Proposals', function () {
  it('Snapshot ', () => {
    const tree = TestRenderer.create(
        <Proposals />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
