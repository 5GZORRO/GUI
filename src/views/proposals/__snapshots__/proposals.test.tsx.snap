// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Proposals Snapshot  1`] = `
<h1>
  Proposals
</h1>
`;
